jwst/datamodels/schemas/dark.schema.yaml

```diff
@@ -1,6 +1,9 @@
 title: Dark current data model
 allOf:
 - $ref: referencefile.schema.yaml
+- $ref: keyword_filter.schema.yaml
+- $ref: keyword_exptype.schema.yaml
```

Comment:
That might be the best approach. We can have you and/or I present the basics of what needs to be done (i.e. teach everyone about using the new keyword_xxxx.schema.yaml files) and then let them dive into updating an assigned set of schemas.